content/blog/asyncapi_codegen_scst.md

```diff
@@ -0,0 +1,136 @@
+---
```

Comment:
please add
```
cover: /images/posts/asyncapi-github-action.webp
weight: 100
```

for cover choose some image that you want to be visible as a cover on landing pages. I recommend using this service https://unsplash.com/s/photos/generate-code but then please remember to add a note panel at the bottom of the post where the image is from